js/app.js

```diff
@@ -145,7 +152,8 @@ angular.module('PathOfDamage', [])
     var section = $scope.sections[sectionKey];
     Object.keys(section.tables).forEach(function (tableKey) {
       var table = section.tables[tableKey];
-      table.totalCalc = table.totalCalc || DataService.additive;
+      table.calcTotal = table.calcTotal || DataService.additive;
```

Comment:
We should probably rename DataService.additive and multiplicative to be more descriptive, like additiveTotal or something